docs/concepts/model-training.md

```diff
@@ -0,0 +1,34 @@
+(model-training)=
+# Model Training
+
+There are many methods and techniques for training a machine learning model. However, there is much more to model training than the training code itself.
+
+MLRun provides MLOps functionality for model training including job orchestration, experiment tracking, creating reusable components, and distributed training out of the box. Please see below for information on each of these capabilies:
+
+### Jobs
+
+Training a model is an example of a `Job` - something that is run once to completion. MLRun provides you with a convenient syntax to take a Python training script and automatically package and deploy on top of a production ready Kubernetes cluster. You can also configure many aspects of this deployment including CPU/MEM/GPU resources, Python dependencies, distributed runtimes, and more.
```

Comment:
Training a model is an example of a `Job` &mdash; something that is run once to completion. MLRun provides you with a convenient syntax to take a Python training script and automatically package and deploy it on top of a production-ready Kubernetes cluster. You can also configure many aspects of this deployment including [CPU/MEM/GPU resources](runtimes/configuring-job-resources.html), Python dependencies, [distributed runtimes](..runtimes/distributed.html), and more.